### ISSUE TITLE
In-dialog SIP INVITE breaks the call session

### ISSUE THREAD

**Issue by denyspozniak** [livekit/sip#96]
Hello!
When sending an in-dialog SIP INVITE (in my case this is placing on hold), the livekit sip gw terminates the session.
If you need additional info (sip pcap traces), I can provide them.

**Comment by dennwc** [comment]
Yes, PCAP will be helpful, thank you. Also, what software did you use to test it?

**Comment by denyspozniak** [comment]
[livekit_re_invite_pcap.zip](https://github.com/livekit/sip/files/15418186/livekit_re_invite_pcap.zip)

It is a PhonerLite sip phone.

**Comment by denyspozniak** [comment]
[livekit_re_inv.zip](https://github.com/livekit/sip/files/15418481/livekit_re_inv.zip)
It is a PhonerLite sip client

**Comment by dennwc** [comment]
Should be fixed by https://github.com/livekit/sip/pull/480
